src/log.c

```diff
@@ -489,6 +489,35 @@ void log_discard(void) {
   syslog_discard = TRUE;
 }
 
+char *log_time(char *buf, int max) {
+      pool *tmp_pool;
```

Comment:
I generally prefer 2-space indentation (and no tabs); PRs are more likely to be accepted if they are consistent with the coding style of the rest of the codebase.